src/core/src/model.cpp

```diff
@@ -34,6 +34,13 @@ BWDCMP_RTTI_DEFINITION(ov::AttributeAdapter<std::shared_ptr<ov::Model>>);
 
 atomic<size_t> ov::Model::m_next_instance_id(0);
 
+namespace ov {
+namespace frontend {
+class FrontEndManager;
+std::shared_ptr<FrontEndManager> get_frontend_manager();
```

Comment:
am I right that because of static frontend manager, we have issue with dynamically loaded OpenVINO.so:
1. FE stores reference to OpenVINO.so
2. OpenVINO stores references to frontend libraries
And because of that, we have cyclic dependencies and libraries are not unloaded?

Reply:
yes, it is the reason that why libopenvino_ir_frontend.so cannot be unloaded. But for libopenvino.so unload issue, there are more static variables and other dependencies/reference.

Reply:
There are quit a lot such datas, not only static variable but also dependences data, I have ever setup a debug environment but not save the data, maybe I can re-setup and share the data to you later.

Reply:
@ilya-lavrenov I sent some details info to you by email, please check it, thanks!